test/index.test.ts

```diff
@@ -195,6 +198,17 @@ describe('featurePolicy', () => {
     }));
   });
 
+  it('can set all values in the allowlist to "*"', () => {
```

Comment:
Could you change these spec titles to something unique, or put them in a `describe('with dashed key names')` block?